packages/twenty-front/src/modules/apollo/services/apollo.factory.ts

```diff
@@ -251,6 +250,7 @@ export class ApolloFactory<TCacheShape> implements ApolloManager<TCacheShape> {
           isDebugMode ? logger : null,
           retryLink,
           streamingRestLink,
+          restFileUploadLink,
```

Comment:
In a previous PR I had asked if you could rename httpLink to uploadLink ; this new restFileUploadLink seems very similar to the httpLink, what justifies creating a new one?